### ISSUE TITLE
Boba Network Bridge Fees Are Too Expensive

### ISSUE THREAD

**Issue by ericyoung142** [omgnetwork/optimism#546]
Used the Fast Bridge to L2 Boba Network, at the time Ethereum average 60 gwei price.  It cost me 2 x $45 transactions, and unable to edit gas fees without transactions failing.  Recently bridged from Ethereum to the Avalanche blockchain for <$10 total, similar gwei price.  I have a faint understanding that L2's require more complex code that L1's, and maybe Avax is compromising security for a lower bridging costs.  Still, the cost of your bridge would make anyone except for the ultra Boba bull balk.  If there's anyway to decrease the complexity of these bridge contracts I think it would go a long way for mass adoption of the Boba Network.

**Comment by CAPtheorem** [comment]
Yes, contract complexity and gas optimizations are on our radar. Expect progress in this regard by the end of the month.

**Comment by CAPtheorem** [comment]
Update - here are some numbers. 

At current (Unix time: 16331455) gas prices (72 Gwei), the costs to bridge to L2 were measured to be:

**Classical Bridge** 0.0032 ETH for approval, 0.0371 ETH to bridge = 0.0403 ETH total cost
**Fast Bridge**      0.0025 ETH for approval, 0.0131 ETH to bridge = 0.0156 ETH total cost

Generally the Fast Bridge is about 1/2 the price of the classical deposit, despite the 0.1% fast swap fee which is paid out to the liquidity providers.

**Comment by CAPtheorem** [comment]
Here is a recent article on cost comparison of Arbitrum vs. Optimism for various bridging and transfer operations:

https://inf.news/en/tech/bc8c0fa2ea6e0e2f3fe8963205cc8a36.html
